.changeset/config.json

```diff
@@ -5,9 +5,9 @@
   "//": "CAUTION: When updating the fields below, you should also update README accordingly.",
   "fixed": [
     [
+      "@logto/core",
```

Comment:
add a comment to explain why it should stay on the top?

Reply:
Cannot add comment in json.